src/service/localization/strings.js

```diff
@@ -104,7 +104,7 @@ export const LocalizedStringId_Enum = {
   AMP_STORY_SHOPPING_CTA_LABEL: '101',
   AMP_STORY_SHOPPING_PLP_HEADER: '102',
   AMP_STORY_SHOPPING_ATTACHMENT_CTA_LABEL: '103',
-  AMP_STORY_SHOPPING_ATTACHMENT_REVIEWS_LABEL: '104',
+  // DEPRECATED_AMP_STORY_SHOPPING_ATTACHMENT_REVIEWS_LABEL: '104',
```

Comment:
Can you remove the strings from all files? I think @mszylkowski has a python script

Reply:
It was only in `en.json`. Removed!